CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log
+
+## [v2.0.0](https://github.com/PolymerLabs/dom5/tree/v2.0.0) (2016-09-23)
+- Upgraded to parse5 v2.2.1
+- (*breaking*) Removed `parse()`, `parseFragment()` and `serialize()`. Use
+  `parse5.parse()`, etc., instead.
+- Nodes can be appended to parents with a `null` `childNodes` array.
+- DocumentFragments that are appended have their `childNodes` array cleared,
+  rather than getting a new empty `childNodes` instance.
```

Comment:
Mention the most important change, that the walk* family of functions will no longer descend into <template> elements.

Also, will they go up out of <template> elements?